### PR TITLE
fix: Remove FQDN and $edit

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/zh-cn/web/api/cssstylesheet/insertrule/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ slug: Web/API/CSSStyleSheet/insertRule
 
 {{ APIRef("CSSOM") }}
 
-**`CSSStyleSheet.insertRule()` **方法用来给当前样式表插入新的样式规则（[CSS rule](/zh-CN/docs/Web/API/CSSRule)），并且包含一些[限制](/zh-CN/docs/Web/API/CSSStyleSheet/insertRule$edit#Restrictions)。
+**`CSSStyleSheet.insertRule()` **方法用来给当前样式表插入新的样式规则（[CSS rule](/zh-CN/docs/Web/API/CSSRule)），并且包含一些[限制](/zh-CN/docs/Web/API/CSSStyleSheet/insertRule#Restrictions)。
```
Comment (yin1999) [inline files/zh-cn/web/api/cssstylesheet/insertrule/index.md:8]:
```suggestion
**`CSSStyleSheet.insertRule()` **方法用来给当前样式表插入新的样式规则（[CSS rule](/zh-CN/docs/Web/API/CSSRule)），并且包含一些[限制](#限制)。
```

File: files/zh-cn/web/api/cssstylesheet/insertrule/index.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ stylesheet.insertRule(rule [, index])
     - **[at-rules](/zh-CN/docs/Web/CSS/At-rule) 类型**（以 `@` 开头的规则，如 `@import, @media` 等）**，**需要 at-identifier 和规则内容。
 
 - `index` {{optional_inline}}
-  - : 一个小于或等于 `stylesheet.cssRules.length` 的正整数，表示新插入的规则在`{{domxref("CSSStyleSheet", "", "", "1")}}.cssRules` 中的位置。默认值是 `0`。（在过去的实现中，这个参数是必需的，详情参见[浏览器兼容性](/zh-CN/docs/Web/API/CSSStyleSheet/insertRule$edit#Browser_compatibility)。)
+  - : 一个小于或等于 `stylesheet.cssRules.length` 的正整数，表示新插入的规则在`{{domxref("CSSStyleSheet", "", "", "1")}}.cssRules` 中的位置。默认值是 `0`。（在过去的实现中，这个参数是必需的，详情参见[浏览器兼容性](/zh-CN/docs/Web/API/CSSStyleSheet/insertRule#Browser_compatibility)。)
```
Comment (yin1999) [inline files/zh-cn/web/api/cssstylesheet/insertrule/index.md:28]:
```suggestion
  - : 一个小于或等于 `stylesheet.cssRules.length` 的正整数，表示新插入的规则在`{{domxref("CSSStyleSheet", "", "", "1")}}.cssRules` 中的位置。默认值是 `0`。（在过去的实现中，这个参数是必需的，详情参见[浏览器兼容性](#浏览器兼容性)。）
```

**Review by yin1999 — APPROVED** [review]
Thanks @nschonni

File: files/zh-cn/web/api/element/firstelementchild/index.md [hunk 3]
```diff
@@ -66,7 +66,5 @@ console.log(foo.firstElementChild.textContent);
 
 ## 参见
 
-## [Ed](/zh-CN/docs/Web/API/Element/lastElementChild$edit#参见)
```
Comment (nschonni) [inline files/zh-cn/web/api/element/firstelementchild/index.md:69]: My original edit was bad, but I think the link was supposed to be in the section if i look at other languages

Comment (yin1999) [reply files/zh-cn/web/api/element/firstelementchild/index.md:69]: It's still a good PR, thanks for your help.

**Pull request merged**